src/__tests__/SignerAlgorithm-test.ts

```diff
@@ -1,14 +1,14 @@
 import SignerAlgorithm from '../SignerAlgorithm'
 import { toSignatureObject } from '../VerifierAlgorithm'
-import SimpleSigner from '../SimpleSigner'
-import EllipticSigner from '../EllipticSigner'
-import NaclSigner from '../NaclSigner'
+import SimpleSigner from '../signers/SimpleSigner'
+import EllipticSigner from '../signers/EllipticSigner'
+import NaclSigner from '../signers/NaclSigner'
 import { ec as EC } from 'elliptic'
 import nacl from 'tweetnacl'
 import { base64ToBytes, base64urlToBytes, stringToBytes } from '../util'
 import { sha256 } from '../Digest'
 const secp256k1 = new EC('secp256k1')
-const privateKey = '278a5de700e29faae8e40e366ec5012b5ec63d36ec77e8a241154cc1d25383f'
+const privateKey = '278a5de700e29faae8e40e366ec5012b5ec63d36ec77e8a2417154cc1d25383f'
```

Comment:
Really trying to understand what changed here & with the signature.

Reply:
Is the change `a241154c` -> `a2417154c` intentional?

Reply:
Yes, and thank you for raising this.

This key was not a proper hex encoding of 32 bytes.
The `SimpleSigner` used BigNumber when importing the privateKey which accepted the nonstandard encoding but generally the 32 bytes encoding in hex should be represented as 64 hex characters.

The reason I changed the key like this and why I didn't just add a leading zero is that this key was already used in other tests and could cause unnecessary confusion for others as it had for me.
It probably got copied wrong at some point.

I can revert this and add a leading zero to maintain the same signatures as a demonstration of equivalence.

Reply:
fixed.
I reverted the key(and added the leading zero) and the signatures.